ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireAssertion.h

```diff
@@ -28,6 +28,7 @@
                useAccessToken:(BOOL)useAccessToken
                 samlAssertion:(NSString*)samlAssertion
                 assertionType:(ADAssertionType)assertionType
+           retryIfServerError:(BOOL)retryIfServerError
```

Comment:
Make this a property on the request object itself. I'm trying to minimize the amount of method parameters if at all possible.